### PR TITLE
Payment svc 106

### PR DESCRIPTION
**Pull request by Grant-Steinfeld**
Hi @MaxShapiro32 , the Payment schema is at a good point, and was tested on postgres db via pgcli

### CODE REVIEW

File: src/payment/scripts/db-schema.sql [hunk 1]
```diff
@@ -1 +1,55 @@
-CREATE TABLE users
+CREATE TABLE tUser (
+	UserID serial PRIMARY KEY,
+	LastName VARCHAR(64) NOT NULL,
+	FirstName VARCHAR(64) NOT NULL,
+	Address VARCHAR(255),
+	City VARCHAR(255),
+	Email VARCHAR(255) UNIQUE NOT NULL,
+	CreditCard VARCHAR(40),
+	created_on TIMESTAMP NOT NULL
```
Comment (MaxShapiro32) [inline src/payment/scripts/db-schema.sql:9]: Is the `created_on` timestamp needed for every table?

File: src/payment/scripts/db-schema.sql [hunk 2]
```diff
@@ -1 +1,55 @@
-CREATE TABLE users
+CREATE TABLE tUser (
+	UserID serial PRIMARY KEY,
+	LastName VARCHAR(64) NOT NULL,
+	FirstName VARCHAR(64) NOT NULL,
+	Address VARCHAR(255),
+	City VARCHAR(255),
+	Email VARCHAR(255) UNIQUE NOT NULL,
```
Comment (MaxShapiro32) [inline src/payment/scripts/db-schema.sql:7]: Would having this unique cause an issue if a user makes multiple purchases with different credit cards but the same email address?

File: src/payment/scripts/db-schema.sql [hunk 3]
```diff
@@ -1 +1,55 @@
...
+	costUSD decimal NOT NULL,
+	created_on TIMESTAMP NOT NULL
+
+);
+
+CREATE TABLE tHotel (
+	HotelID serial PRIMARY KEY,
+	ReservationUUID uuid NOT NULL,
+	HotelName VARCHAR(255),
+	locationAddress VARCHAR(255),
```
Comment (MaxShapiro32) [inline src/payment/scripts/db-schema.sql:40]: Currently we have city and country for location data. I am wondering if this should be separated or just put both in here separated by a comma

File: src/payment/scripts/db-schema.sql (hunk 1 above)
Comment (Grant-Steinfeld) [reply src/payment/scripts/db-schema.sql:9]: definitely user and reservation table, let me think about the others

File: src/payment/scripts/db-schema.sql (hunk 2 above)
Comment (Grant-Steinfeld) [reply src/payment/scripts/db-schema.sql:7]: ah - good point then I'll make it a compund unique constraint on email and creditcard

File: src/payment/scripts/db-schema.sql (hunk 3 above)
Comment (Grant-Steinfeld) [reply src/payment/scripts/db-schema.sql:40]: yes this was draft one, location will be city and country 
should we include street address ( like 43 Main Stree ) ?

Comment (MaxShapiro32) [reply src/payment/scripts/db-schema.sql:40]: @Grant-Steinfeld We don't currently have addresses so I would say hold off on that

Comment (Grant-Steinfeld) [reply src/payment/scripts/db-schema.sql:40]: OK - merged as I had some new stuff to PR - oddly could not do that from my Fork?

**Pull request merged**